plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -1058,7 +1142,8 @@ int create_log_group(struct flb_cloudwatch *ctx)
             ctx->group_created = FLB_TRUE;
             flb_sds_destroy(body);
             flb_http_client_destroy(c);
-            return 0;
+            int ret = set_log_group_retention(ctx);
```

Comment:
always declare variables at the top of the function, per the style guide: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#variable-definitions